tests/test_unittest/test_gtscript_frontend.py

```diff
@@ -293,6 +293,50 @@ def definition_func(inout_field: gtscript.Field[float]):
             definition_func, name=inspect.stack()[0][3], module=self.__class__.__name__
         )
 
+    def test_use_in_expr_multiple_return(self):
+        @gtscript.function
+        def func():
+            tmp1 = 1
+            tmp2 = 2
+            return tmp1, tmp2
+
+        def definition_func(inout_field: gtscript.Field[float]):
+            from gt4py.__gtscript__ import PARALLEL, computation, interval
+
+            with computation(PARALLEL), interval(...):
+                inout_field = func() + 1
+
+        with pytest.raises(
+            gt_frontend.GTScriptSyntaxError, match="Function returning tuple used in expression."
+        ):
+            parse_definition(
+                definition_func, name=inspect.stack()[0][3], module=self.__class__.__name__
+            )
+
+    def test_use_in_call_arg_multiple_return(self):
+        @gtscript.function
+        def func():
+            tmp1 = 1
+            tmp2 = 2
+            return tmp1, tmp2
+
+        @gtscript.function
+        def func_outer(arg):
+            return arg + 1
+
+        def definition_func(inout_field: gtscript.Field[float]):
+            from gt4py.__gtscript__ import PARALLEL, computation, interval
+
+            with computation(PARALLEL), interval(...):
+                inout_field = func_outer(func())
+
+        with pytest.raises(
+            gt_frontend.GTScriptSyntaxError, match="Function returning tuple used in expression."
+        ):
+            parse_definition(
+                definition_func, name=inspect.stack()[0][3], module=self.__class__.__name__
+            )
```

Comment:
This test sort of makes my brain hurt. Could we disallow multiple return values being used in a function call? I'd rather see the regular Python syntax here `inout_field = func_outer(*func())` if `func` has multiple return values.

What do you think?